degirum_tools/notifier.py

```diff
@@ -66,6 +76,19 @@ def __init__(
         self._annotation_color = annotation_color
         self._annotation_corner = annotation_corner
         self._annotation_cool_down = annotation_cool_down
+        self._tag = tag
+        
+        # Setting up apprise object and config if the user provides a path to a config file
+        self._apprise_obj = None
+        if self._token:
+            self._apprise_obj = Apprise()
+            conf = AppriseConfig()
+            conf.add(self._token)
+            self._apprise_obj.add(conf)
+
+            for entry in iter(self._apprise_obj):
+                print(entry)

```

Comment:
We usually don't print anything to stdout. Because we are library providers. We don't want to contaminate stout of client's app with our internal debug info

Reply:
Good to know. For future reference, how would I properly log things for internal debugging?